homotopy.tex

```diff
@@ -1153,12 +1153,14 @@ \section{Fiber sequences and the long exact sequence}
 \begin{cor}\label{thm:conn-pik}
   Let $f:A\to B$ be $n$-connected and $a:A$, and define $b\defeq f(a)$.  Then:
   \begin{enumerate}
-  \item If $k\le n$, then $\pi_k(f):\pi_k(A,a) \to \pi_k(B,b)$ is an isomorphism.
-  \item If $k=n+1$, then $\pi_k(f):\pi_k(A,a) \to \pi_k(B,b)$ is surjective.
+  \item If $k\le n$, then $\pi_k(f):\pi_k(A,a) \to \pi_k(B,b)$ is an isomorphism.\label{item:conn-pik1}
+  \item If $k=n+1$, then $\pi_k(f):\pi_k(A,a) \to \pi_k(B,b)$ is surjective.\label{item:conn-pik2}
   \end{enumerate}
 \end{cor}
 \begin{proof}
-  As part of the long exact sequence, for each $k$ we have an exact sequence
+  For $k=0$, part~\ref{item:conn-pik1} follows from Lemma~\ref{lem:connected-map-equiv-truncation}, noticing that $\pi_0(f)\equiv\trunc 0f$.
+  For $k=0$ part~\ref{item:conn-pik2} follows from Exercise~\ref{ex:is-conn-trunc-functor}, noticing that a function is surjective iff it's $(-1)$-connected, by Lemma \ref{thm:minusoneconn-surjective}.
```

Comment:
Sorry, just noticed something else: we use cleveref, so instead of "Lemma \ref{...}" you can just write "\cref{...}" and the word "Lemma" will be added automatically.  Same for exercises.